lib/scrabble.rb

```diff
@@ -1,4 +1,92 @@
 module Scrabble
+
+LETTER_VALUES = [
+  [1, "A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+  [2, "D", "G"],
+  [3, "B", "C", "M", "P"],
+  [4, "F", "H", "V", "W", "Y"],
+  [5, "K"],
+  [8, "J", "X"],
+  [10, "Q", "Z"]
+]
+
+class Scrabble
   # require your gems and classes here
   # require_relative 'scrabble/whatevs'
+  require "awesome_print"
+
+def self.letter_score(letter)
```

Comment:
All of your methods inside the Scrabble class should be tabbed in rather than inline with the class definition